dashboard/app/models/plc/enrollment_unit_assignment.rb

```diff
@@ -64,8 +64,8 @@ def enroll_user_in_unit_with_learning_modules(learning_modules)
     end
   end
 
-  def focus_area_positions
-    plc_module_assignments.map {|a| a.plc_learning_module.stage.absolute_position unless a.plc_learning_module.required?}.compact
```

Comment:
Presumably a teacher will never have a lockable stage as a focus area?  If so, could we just change this to `relative_position` and leave everything as-is?